docs/1.getting-started/3.configuration.md

```diff
@@ -120,7 +120,7 @@ const appConfig = useAppConfig()
 
 As stated above, `runtimeConfig` and `app.config` are both used to expose variables to the rest of your application. To determine whether you should use one or the other, here are some guidelines:
 
-- `runtimeConfig`: Private or public tokens that need to be specified after build using environment variables.
+- `runtimeConfig` : Private or public tokens that need to be specified after build using environment variables.
 - `app.config` : Public tokens that are determined at build time, website configuration such as theme variant, title and any project config that are not sensitive.
```

Comment:
I'd rather remove the space here I think 🤔

Reply:
Agree (same applies to line 124)